azure/application-gateway/advanced/examples/basic/main.tf

```
@@ -13,13 +16,13 @@ locals {
 
 resource "azurerm_resource_group" "example" {
   name     = "awg-rg"
-  location = local.location
+  location = "uksouth"
```

if new, add tags to resource group?


```suggestion
  tags = local.tags
```